thankyou.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Submission</title>
+    </head>
+    <body>
+        <h1>Thank you for keeping us updated. We hope to hear from you some other time.</h1>
+    </body>
+</html>
```

Comment:
Add a new line after this closing tag

Reply:
i don't understand what you mean by this.

Reply:
After the last character i.e. `>`, just press the enter button.
Feel free to merge if don't get it now, we can go through it during the call later.